### DISCUSSION TITLE
Interpreting BCR rearragnements

### DISCUSSION THREAD

**Discussion by alexsteemers** [milaboratory/mixcr#1478] [Q&A] [answered]
Hi! I have used MIXCR to look at the BCR clonality in my single cell WGS data. I have a single cell that has the following BCR annotation: IGHV3-21,IGHV3-7_IGHD6-19_IGHJ5. I am not sure how to interpret the "IGHV3-21,IGHV3-7" part. Any help will be much appreciated.

**Comment by mizraelson** [comment]
Hi,

Could you please share the command you used along with the output that is causing concern? If I understand your question correctly, you are asking why a certain clone is listed with two V genes in the table. This could be due to insufficient sequencing information to distinguish between the two genes. 

Sincerely,
Mark

**Reply by alexsteemers** [reply]
Hi Mark,

I used the allVHitsWithScore output, and according to previous discussions it seems that using BestVHits is better.

**Reply by mizraelson** [reply, marked as answer]
Well, the difference is that 'allVHitsWithScore' shows all possible V genes along with their alignment scores, while 'bestVHits' shows only the V gene with the highest alignment score.
